roles/openshift-applier/tasks/pre-check.yml

```diff
@@ -55,7 +55,7 @@
       ignore_errors: true
       register: helm_version_check
       when:
-        - openshift_cluster_content | join(', ') | regex_search('helm_chart')
+        - openshift_cluster_content | join(', ') | regex_search('helm')
```

Comment:
@jfilipcz thank you for catching this and submitting the PR. The implementation (as was done previously) is quite risky, however, and perhaps even more so now that the uniqueness of the search word is very low. Hence, I'd like to suggest that we enhance this to do a bit more accurate checking for any dictionary keys named `helm`. The following (or a variant) of this should work. Let me know if you disagree. 

```
openshift_cluster_content | sum(attribute='content', start=[]) | selectattr('helm', 'defined') | list | count
```